cross-reference/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use an official Python runtime as a parent image
+FROM python:3.11-slim-buster
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# Set work directory
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y build-essential mariadb-client libmariadbclient-dev
```

Comment:
probably the mariadb-client and build-essential are not needed. some leftovers from other dockerfiles. I'll remove them and we'll see after testing

Reply:
> hadolint runs

Nope, it does not, it runs on the wrong PATH...

```console
~/buildbot/cross-reference remotes/vlad/crossref_docker ≡
.venv ❯ podman run -it -i -v $(pwd):/mnt -w /mnt ghcr.io/hadolint/hadolint:latest hadolint /mnt/Dockerfile
WARN[0000] "/" is not a shared mount, this could cause issues or missing mounts with rootless containers 
Trying to pull ghcr.io/hadolint/hadolint:latest...
Getting image source signatures
Copying blob db4123164570 done   | 
Copying config da13a5ec2e done   | 
Writing manifest to image destination
/mnt/Dockerfile:12 DL3008 warning: Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
/mnt/Dockerfile:12 DL3009 info: Delete the apt-get lists after installing something
/mnt/Dockerfile:18 DL3013 warning: Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`
```

Reply:
DL3008 are disabled but DL3009 is not.

Reply:
The paths are too cryptic. I am not sure what it wants from `pip`

Reply:
DL3009 is also disabled. Anyway not sure what it wants now